ucrop/src/main/java/com/yalantis/ucrop/UCropFragment.java

```diff
@@ -449,7 +449,10 @@ private void setAngleTextColor(int textColor) {
 
     private void setScaleText(float scale) {
         if (mTextViewScalePercent != null) {
-            mTextViewScalePercent.setText(String.format(Locale.getDefault(), "%d%%", (int) (scale * 100)));
+            float initialScale = mGestureCropImageView.getInitialMinScale();
+            mTextViewScalePercent.setText(
+                String.format(Locale.getDefault(), "%.2f%%", (scale/ initialScale * 100))
```

Comment:
@ashiagr , I would suggest using 

                String.format(Locale.getDefault(), "%d%%", Math.round(scale/ initialScale * 100))

This will prevent incorrect value like 99.99% when it should be 100%. Also integer value should be enough to display. Fractional part is redundant.